<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="CPX-ANXXTNSGFUP" ignored="1" name="Test-class test" tests="8" failures="2" timestamp="14 paÅº 2019 11:19:13 GMT" time="0.076" errors="0">
  <testcase name="testCalculateAppriasal" time="0.019" classname="Test.TestEmployeeDetails"/>
  <testcase name="testPrintMessage" time="0.019" classname="Test.Test1"/>
  <testcase name="testPrintMessage" time="0.019" classname="Test.GroupTestExample"/>
  <testcase name="testSalutationMessage" time="0.018" classname="Test.Test2"/>
  <testcase name="testSalutationMessage" time="0.019" classname="Test.IgnoreTest"/>
  <testcase name="testCalculateYearlySalary" time="0.007" classname="Test.TestEmployeeDetails"/>
  <testcase name="testSalutationMessage" time="0.005" classname="Test.GroupTestExample">
    <failure type="java.lang.AssertionError" message="expected [Hi!.com] but found [tutorialspoint.com]">
      <![CDATA[java.lang.AssertionError: expected [Hi!.com] but found [tutorialspoint.com]
at Test.GroupTestExample.testSalutationMessage(GroupTestExample.java:23)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- testSalutationMessage -->
  <testcase name="testingExitMessage" time="0.002" classname="Test.GroupTestExample">
    <failure type="java.lang.AssertionError" message="expected [www.Hi!.com] but found [www.tutorialspoint.com]">
      <![CDATA[java.lang.AssertionError: expected [www.Hi!.com] but found [www.tutorialspoint.com]
at Test.GroupTestExample.testingExitMessage(GroupTestExample.java:30)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- testingExitMessage -->
  <testcase name="testPrintMessage" time="0.0" classname="Test.IgnoreTest">
    <ignored/>
  </testcase> <!-- testPrintMessage -->
</testsuite> <!-- Test-class test -->
